test/support/samples/swagger_formater.ex

```diff
@@ -1,5 +1,5 @@
-defmodule Xcribe.Support.Samples.SwaggerFormater do
-  def path_item_object_without_request_body do
+defmodule Xcribe.Support.Samples.SwaggerFormater.PathItemObject do
```

Comment:
Could we also rename the file?

`mv test/support/samples/swagger_formater.ex test/support/samples/swagger_formater/path_item_object.ex`

Reply:
Done :P